### ISSUE TITLE
Issue on Prototype Fusion Strategy

### ISSUE THREAD

**Issue by viviGeng** [zhangbq-research/Prototype_Completion_for_FSL#1]
Hi. I tried to reproduce results for MiniImageNet following your instruction, but failed to get the accuracy listed in your paper.
Could you give me some tips? 
Also, where is the Gaussian fusion strategy implemented in the code?

Looking forward to your reply.

**Comment by zhangbq-research** [comment]
I guess this problem should appear in the parameter setting, please check whether the parameter setting of each step is consistent with our instructions. Besides, the Gaussian fusion strategy is only used in meta-training and meta-test phase. In these two steps,  note that the  parameter ``--head'' should be set to ``FuseCosNet'' instead of CosineNet. The detailed codes about the Gaussian fusion strategy can be found at the file of  ``/models/classification_heads.py'' (please see the lines 129 - 155).

**Comment by viviGeng** [comment]
Should I set the parameter **--head** as "FuseCosNet"  instead of CosineNet in metainfer phase? Otherwise, I will fail to load **'best_model_meta_val_{}w_{}s_{}.pth'.format(opt.test_way, opt.val_shot, opt.head)** in metatrain and metatest phase.

**Comment by zhangbq-research** [comment]
Yes, in metainfer phase, the parameter "--head" should also be set to  "FuseCosNet" .
